enos/enos-modules.hcl

```diff
@@ -0,0 +1,399 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+module "autopilot_upgrade_storageconfig" {
+  source = "git::https://github.com/hashicorp/vault.git//enos/modules/autopilot_upgrade_storageconfig?ref=main"
```

Comment:
Should we make the ref configurable for all of these in case we need to pin to a specific sha for any reason?